applications_tests/gls_sharp_navier_stokes_3d/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+IF(DEAL_II_WITH_OPENCASCADE)
+add_subdirectory(opencascade)
+ENDIF()
```

Comment:
So you put the opencascade tests in another folder? Why not use the test identifier? that did not work?

Reply:
I simply copied the structure used in dealii, but maybe there is a better structure.